Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:14-alpine as nodejs
+
+FROM nodejs as builder
+
+WORKDIR /excalidraw-json
+
+ARG S3_ENDPOINT
+ARG S3_ACCESS_KEY_ID
+ARG S3_SECRET_ACCESS_KEY
+ARG S3_BUCKET_NAME="excalidraw"
+ARG S3_FORCE_PATH_STYLE="true"
+ARG ALLOWED_ORIGIN="*"
+
+ENV EXCALIDRAW_S3_ENDPOINT="${S3_ENDPOINT}" \
```

Comment:
Actually on second thoughts.. Are these still configurable once you've built an image?

Reply:
I do not know for sure. I followed same logic that we used for the excalidraw frontend. But I think the excalidraw frontend is fully compiled. I'll do the test locally to see if I can inject the values at runtime. If I can, then we can remove all ARGS and just inject the values at runtime

Reply:
Cool. Yeah my worry is that if this image was built and pushed to DockerHub, then it wouldn't be very useful (if it's not parameterisable).